tests/main/confinement-classic/task.yaml

```diff
@@ -1,10 +1,13 @@
 summary: trivial snap with classic confinement runs correctly
+
+systems: [-ubuntu-core-16-*, -fedora-*, -opensuse-*]
```

Comment:
@zyga Agree with you, in fact I added a lot of comments explaining that in the fedora branch https://github.com/snapcore/snapd/pull/3505), and the idea is that when I merge both branches the comments will be on each task. I already fixed this test to support opensuse btw